base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/dialogs/MultipleConfigsSelectionDialog.java

```diff
@@ -105,30 +106,39 @@ public MultipleConfigsSelectionDialog(
 		return selected;
 	}
 
-	@Override
-	protected void okPressed() {
-        selected = asString(items.getSelection());
-		super.okPressed();
-	}
+    @Override
+   protected void okPressed() {
+       Collection<String> selectedItems = asString(items.getSelection());
+       
+       /* The following ensures that double clicking on the description/white space doesn't
+        * launch the process using a description/null as the name.
+        */    
+       List<String> names = (List<String>) ConfigInfo.names(available);
```

Comment:
You don't need the cast here
```suggestion
       Collection<String> names = ConfigInfo.names(available);
```